src/containers/JoinExit.js

```diff
@@ -100,8 +99,9 @@ class JoinExitContainer extends React.Component {
         const web3 = store.get('web3')
         const isSignedIn = walletAddress && walletAddress.length
 
-      const canJoin = joinAmount && (Number(joinAmount) <= Number(daiBalanceDecimal))
-      const canExit = exitAmount && (Number(exitAmount) <= Number(chaiBalanceDecimal))
+        const canJoin = joinAmount && (joinAmount <= daiBalanceDecimal)
```

Comment:
is there any reason you would like to use a lexicographical ordering of the strings of the numbers rather than comparing the numbers they represent?

Reply:
oh, I see. You're thinking that `daiBalanceDecimal` will always be a number. The problem is that `joinAmount` will most of the time be a string. I suggest we consistently use strings for store vars

Reply:
My previous change also turned joinAmount into a Decimal, not a string which should make it a proper numerical comparison. It's possible that you reverted that change again?

Reply:
Likely as it wasnt converting it to string before the web3.toWei call